website/source/docs/guides/monitoring-telegraf.html.md

```diff
@@ -0,0 +1,315 @@
+with each metric. Tags can be used by Grafana to filter data on your dashboards (for example, displaying only
+the data for which `role=consul-server`. Telegraf is compatible with the DogStatsD format and allows us to add
+our own tags too, as you'll see later.
+
+The second option tells Consul not to insert the hostname in the names of the metrics it sends to statsd, since the hostnames will be sent as tags. Without this option, the single metric `consul.raft.apply` would become multiple metrics:
+
+        consul.server1.raft.apply
+        consul.server2.raft.apply
+        consul.server3.raft.apply
+
+If you are using a different agent (e.g. Circonus, Statsite, or plain statsd), you can find the configuration reference [here][consul-telemetry-config].
+
+[tagging]: https://docs.datadoghq.com/getting_started/tagging/
+[consul-telemetry-config]: https://www.consul.io/docs/agent/options.html#telemetry
+[consul-telemetry-ref]: https://www.consul.io/docs/agent/telemetry.html
+[telegraf-input-plugins]: https://docs.influxdata.com/telegraf/v1.6/plugins/inputs/
+
+## Important Metric Aggregates and Alerting
+
+### Transaction timing
+
+| Metric Name              | Description |
+| :----------------------- | :---------- |
+| `consul.kvs.apply`       | This measures the time it takes to complete an update to the KV store. |
+| `consul.txn.apply`       | This measures the time spent applying a transaction operation. |
+| `consul.raft.apply`      | This counts the number of Raft transactions occurring over the interval. |
+| `consul.raft.commitTime` | This measures the time it takes to commit a new entry to the Raft log on the leader. |
+
+**Why they're important:** Taken together, these metrics indicate how long it takes to complete write operations
+in various parts of the Consul cluster. Generally these should all be fairly consistent and no more than a few
+milliseconds. Sudden changes in any of the timing values could be due to unexpected load on the Consul servers, or
+due to problems on the servers themselves.
+
+**What to look for:** Deviations (in any of these metrics) of more than 50% from baseline over the previous hour.
+
+### Leadership changes
+
+| Metric Name | Description |
+| :---------- | :---------- |
+| `consul.raft.leader.lastContact` | Measures the time since the leader was last able to contact the follower nodes when checking its leader lease. |
+| `consul.raft.state.candidate` | This increments whenever a Consul server starts an election. |
+| `consul.raft.state.leader` | This increments whenever a Consul server becomes a leader. |
+
+**Why they're important:** Normally, your Consul cluster should have a stable leader. If there are frequent elections or leadership changes, it would likely indicate network issues between the Consul servers, or that the Consul servers themselves are unable to keep up with the load.
+
+**What to look for:** If `candidate` > 0, or `leader` > 0, or `lastContact` greater than 200ms.
```

Comment:
This is a mixture of good and bad signs - candidate should be 0 but leader > 0 is a healthy state (at least assuming it's 1. I suggest changing to `leader` != 1 maybe or just being more explicit that these are signs of unhealthy/healthy behaviour.